pkgs/development/libraries/libvirt/default.nix

```diff
@@ -76,7 +78,7 @@ in stdenv.mkDerivation rec {
   ] ++ optionals enableGlusterfs [
     glusterfs
   ] ++ optionals stdenv.isDarwin [
-    libiconv gmp
+    libiconv gmp gnumake Carbon AppKit
```

Comment:
Isn't gnumake part of the stdenv?

Reply:
Could be. libvirt expects gnumake as gmake. Added this as debugging.